nengo/builder/connection.py

```diff
@@ -169,7 +171,7 @@ def get_prepost_signal(is_pre):
     # Sample transform if given a distribution
     transform = (conn.transform.sample(conn.size_out, conn.size_mid, rng=rng)
                  if isinstance(conn.transform, Distribution) else
-                 conn.transform)
+                 np.array(conn.transform))
```

Comment:
The only reason to make a copy here is so that if someone accesses `BuiltConnection.transform` it isn't aliased with the `Connection.transform` right?  Just wondering if we could mark it as readonly or something instead, to cut down on memory usage (since these could be large arrays).